<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Process</class>
 <widget class="QWidget" name="Process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>876</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Load</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Train dataset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonLoadTrainingData">
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditTrainingData"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Test dataset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonLoadTestingData">
                  <property name="text">
                   <string>Load</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEditTestingData"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupNormalization">
         <property name="title">
          <string>Normalization</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkNormalizeUnit">
              <property name="text">
               <string>Normalize to unit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkNormalizeZeroCenter">
              <property name="text">
               <string>Normalize to 0-center</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkNormalizeUnitWithZeroCenter">
              <property name="text">
               <string>Normalize to unit with 0-center</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkNormalizationAll">
              <property name="text">
               <string>All Normalization</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPreprocess">
         <property name="title">
          <string>Preprocess</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkRemoveSimilarFeatures">
            <property name="text">
             <string>PearsonCorrelationCoefficients</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkPCA">
            <property name="text">
             <string>PCA</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkPreprocessAll">
            <property name="text">
             <string>All Preprocess</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSelector">
         <property name="title">
          <string>Feature Selector</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>minNumber</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMinFeatureNumber">
                <property name="minimum">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>maxNumber</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBoxMaxFeatureNumber">
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>20</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QCheckBox" name="checkANOVA">
                <property name="text">
                 <string>ANOVA</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkRFE">
                <property name="text">
                 <string>RFE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkRelief">
                <property name="text">
                 <string>Relief</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkMRMR">
                <property name="text">
                 <string>mRMR</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkFeatureSelectorAll">
                <property name="text">
                 <string>All Feature Selector</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupClassifier">
         <property name="title">
          <string>Classifier</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QCheckBox" name="checkSVM">
              <property name="text">
               <string>SVM</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="checkAE">
              <property name="text">
               <string>AE</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QCheckBox" name="checkRF">
              <property name="text">
               <string>Random Forest</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QCheckBox" name="checkDecisionTree">
              <property name="text">
               <string>Decision Tree</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QCheckBox" name="checkLRLasso">
              <property name="text">
               <string>LR-Lasso</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkLDA">
              <property name="text">
               <string>LDA</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QCheckBox" name="checkNaiveBayes">
              <property name="text">
               <string>Naive Bayes</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="checkLogisticRegression">
              <property name="text">
               <string>Logistic Regression</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="checkGaussianProcess">
              <property name="text">
               <string>Gaussian Process</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="checkAdaboost">
              <property name="text">
               <string>Adaboost</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="checkClassifierAll">
              <property name="text">
               <string>All Classifier</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QCheckBox" name="checkHyperParameters">
              <property name="text">
               <string>Hyper-Parameters?</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Cross Validation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QRadioButton" name="radio5folder">
              <property name="text">
               <string>5-Folder</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio10Folder">
              <property name="text">
               <string>10-Folder</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioLOO">
              <property name="text">
               <string>LeaveOneOut</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRun">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Run and Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pipeline Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listOnePipeline"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditDescription"/>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Verbose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="textEditVerbose"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
